lib/scholarsphere/redis_config.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Scholarsphere
+  class RedisConfig
+    def call
+      redis_config
+    end
+
+    def redis_host
```

Comment:
@whereismyjetpack I think all the `redis_` bits are unnecessary since it's the RedisConfig class, right?